packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -125,12 +125,17 @@ abstract class CameraPlatform extends PlatformInterface {
 
   /// Starts a video recording.
   ///
+  /// If streamCallback is passed then the video frames will also be streamed.
+  ///
   /// The length of the recording can be limited by specifying the [maxVideoDuration].
   /// By default no maximum duration is specified,
   /// meaning the recording will continue until manually stopped.
   /// With [maxVideoDuration] set the video is returned in a [VideoRecordedEvent]
   /// through the [onVideoRecordedEvent] stream when the set duration is reached.
-  Future<void> startVideoRecording(int cameraId, {Duration? maxVideoDuration}) {
+  Future<void> startVideoRecording(int cameraId,
```

Comment:
This is a breaking change, so can't be made this way. Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters

(Sorry @bparrishMines I meant to tag you in for app-facing + platform interface, not just for app-facing.)

Reply:
Cool, I [figured this was the case](https://github.com/flutter/plugins/pull/6290#issuecomment-1220092690) but was just waiting for some confirmation. I'll split this out into a separate PR and then update this one accordingly.

Reply:
PR created: https://github.com/flutter/plugins/pull/6550

Reply:
This has been merged now and this PR updated accordingly.